test/infer_rt_test.py

```diff
@@ -226,6 +229,20 @@ def test_generate_infection_rate_metric_two_aggregate_levels():
     assert "06075" in returned_fips
 
 
+@pytest.mark.slow
+def test_generate_infection_rate_new_orleans_patch():
```

Comment:
It might be worth having a test that makes sure that the patch is calculated for the full set even if only requesting a subset of region.

Although if that involves mocking up some dataclasses that we don't already have structure around, then I'm happy with just adding a warning comment in the patch code noting the intent.

Reply:
I don't think it is worth complicating the system right now to support a case that isn't expected in production. It'd be nice to run one fips and produce the same value but for now just trying to retire all these intermediate files.